docs/docs/08.1-more-about-refs.md

```diff
@@ -1,75 +1,76 @@
 ---
 id: more-about-refs
-title: More About Refs
+title: Refs to Components
 permalink: more-about-refs.html
 prev: working-with-the-browser.html
 next: tooling-integration.html
 ---
-After returning the structure of your UI from the render method, you may find yourself wanting to "reach out" and invoke methods on component instances returned from `render()`. Often, doing something like this isn't necessary for making data flow through your application, because the Reactive data flow always ensures that the most recent `props` are sent to each child that is output from `render()`. However, there are a few cases where it still might be necessary or beneficial.
+After building your component, you may find yourself wanting to "reach out" and invoke methods on component instances returned from `render()`.  In most cases, this is ideally unnecessary (and should be avoided) because the reactive data flow always ensures that the most recent props are sent to each child that is output from render().  However, there are a few cases where it still might be necessary or beneficial, so React provides an escape hatch known as `refs`.  These `refs` (references) are especially useful when you need to: find the DOM markup rendered by a component (for instance, to position it absolutely), use React components in a larger non-React application, or transition your existing codebase to React.
 
-Consider the case, when you wish to tell an `<input />` element (that exists within your instances sub-hierarchy) to focus after you update its value to be the empty string, `''`.
+Let's look at how to get a ref, and then dive into a complete example.
 
-```javascript
-  var App = React.createClass({
-    getInitialState: function() {
-      return {userInput: ''};
-    },
-    handleChange: function(e) {
-      this.setState({userInput: e.target.value});
-    },
-    clearAndFocusInput: function() {
-      this.setState({userInput: ''}); // Clear the input
-      // We wish to focus the <input /> now!
-    },
-    render: function() {
-      return (
-        <div>
-          <div onClick={this.clearAndFocusInput}>
-            Click to Focus and Reset
-          </div>
-          <input
-            value={this.state.userInput}
-            onChange={this.handleChange}
-          />
-        </div>
-      );
-    }
-  });
+## The ref returned from React.render
+
+No to be confused with the `render()` function that you define on your component (and which returns a virtual DOM element), [React.render()](/react/docs/top-level-api.html#react.render) will return a reference to your component's **backing instance** (or null for [stateless components](/react/docs/reusable-components.html#stateless-functions)).
```

Comment:
Thanks for catching typo!

With regards to React.render -> ReactDOM.render, I think we should hold off on that change since the link destination says React.render() and we should be consistent to avoid confusion.